packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx

```diff
@@ -51,7 +51,7 @@ const ResourceViewActionMenu: FC<ResourceViewActionMenuProps> = ({
     const location = useLocation();
     const { projectUuid } = useParams<{ projectUuid: string }>();
     const organizationUuid = user.data?.organizationUuid;
-    const { data: spaces = [] } = useSpaceSummaries(projectUuid);
+    const { data: spaces = [] } = useSpaceSummaries(projectUuid, true);
```

Comment:
shouldn't `true` be instead `user.data.ability.can(something-that-indicates-the-user-is-admin)`?

Reply:
@joaoviana that part is handled on the backend